lib/model/frames/form.js

```diff
@@ -50,10 +51,10 @@ class Form extends Frame.define(
   table('forms'),
   'id',                                 'projectId',    readable,
   'xmlFormId',    readable,             'state',        readable, writable,
-  'name',         readable,             'currentDefId',
```

Comment:
i'd probably rather just name get taken out here and the rest left alone to preserve the line history.

Reply:
done!